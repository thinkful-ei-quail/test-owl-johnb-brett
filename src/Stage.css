.stage {
  background-color: blue;
}
.observing {
  display: none;
}
