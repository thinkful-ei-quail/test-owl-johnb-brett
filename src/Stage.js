import React from "react";
import Participant from "./Participant";
import data from "./data";

function Stage(props) {
  return (
    <div className="stage">
      <div className="other">
        <Participant />
        <div className="avatar"></div>
      </div>

      <div className="self">
        <Participant />
      </div>
    </div>
  );
}

export default Stage;
