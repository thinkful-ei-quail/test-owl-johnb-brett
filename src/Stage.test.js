import React from "react";
import ReactDOM from "react-dom";
import Stage from "./Stage";
import renderer from "react-test-renderer";

it("should render without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Stage />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer.create(<Stage />).toJSON();
  expect(tree).toMatchSnapshot();
});
