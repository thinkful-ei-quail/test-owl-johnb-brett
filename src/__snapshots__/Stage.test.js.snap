// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="stage"
>
  <div
    className="other"
  >
    <div>
      <img
        alt="avatar"
      />
      <h3 />
      <span
        style={
          Object {
            "color": "lightgrey",
            "fontSize": 50,
          }
        }
      >
        •
      </span>
      <span>
        observing
      </span>
    </div>
    <div
      className="avatar"
    />
  </div>
  <div
    className="self"
  >
    <div>
      <img
        alt="avatar"
      />
      <h3 />
      <span
        style={
          Object {
            "color": "lightgrey",
            "fontSize": 50,
          }
        }
      >
        •
      </span>
      <span>
        observing
      </span>
    </div>
  </div>
</div>
`;
